commands/export.go

```diff
@@ -21,6 +21,10 @@ func init() {
 }
 
 func Export(args []string, flags *pflag.FlagSet) error {
+	if !utils.Confirm("Do you really want to dump your whole pick safe?", false) {
+		return errors.New("Aborted as requested")
```

Comment:
Regarding the pipe to `jq`- currently you cannot pipeline pick because it requires interactive auth; do you have a workaround you're using?